applications/nrf5340_audio/README.rst

```diff
@@ -685,6 +685,7 @@ To indicate the tasks performed, the application uses the LED behavior described
 |                          | Blinking blue - Depending on the device and the mode:                                               |
 |                          |                                                                                                     |
 |                          | * Headset: Kits have started streaming audio (BIS and CIS modes).                                   |
+|                          |            LEDS will blink in sync to visualize synchronization of audio between kits.              |
```

Comment:
Nitpick: LEDs. We need to discuss on Monday - ref. the seq_num issue. We may have to document differently until that has been resolved.